src/main/java/uk/gov/hmcts/cmc/claimstore/services/OffersService.java

```diff
@@ -41,26 +43,36 @@ public void makeOffer(Claim claim, Offer offer, MadeBy party) {
     }
 
     public void accept(Claim claim, MadeBy party) {
+        assertMediationHasStarted(claim);
         assertSettlementIsNotReached(claim);
 
-        Settlement settlement = claim.getSettlement().orElse(new Settlement());
+        Settlement settlement = claim.getSettlement().get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS3655)